_includes/navbar.html

```diff
@@ -1,79 +1,79 @@
 <header class="header">
-  <div id="navbar-original" hidden>
-    <nav class="navbar navbar-default">
-      <div class="container">
-        <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class="navbar-brand" href="/"><strong>Gruntwork</strong>.io</a>
+    <div id="navbar-original" hidden>
```

Comment:
It looks like there are still a bunch of changes to the navbar too. Did this also not get reverted?

Reply:
It's just whitespaces. Nothing changed